keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/VersioningStrategyComparators.kt

```diff
@@ -31,9 +31,11 @@ import org.slf4j.LoggerFactory
 import org.springframework.util.comparator.NullSafeComparator
 
 val VersioningStrategy.comparator: Comparator<String>
+  // TODO: this needs to support versioning strategies provided by artifact plugins
```

Comment:
Is this TODO going to be implemented in part 3, or is that functionality not coming until later?

Reply:
Part 3.